boards/nxp/mimxrt700_evk/mimxrt700_evk_mimxrt798s_cpu1_defconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_GPIO=y
+CONFIG_PINCTRL=y
```

Comment:
Just so you know, there is an active effort to remove `CONFIG_PINCTRL` at the board level: #79065. If that PR merges first, this one should be updated.